### PR TITLE
style(docs): Use simple <pre> for monospace blocks

### LINKED ISSUE #4051: Gateway log export example
- Administer -> Viewing logs -> Log export

### PR DESCRIPTION
**Pull request by jamilbk**
- Add logs export example, fixes #4051 
- Use brand color palette for `pre` blocks

These have been updated to match our colorscheme.

Code blocks (\`\`\`elixir) will pull in the syntax highlighting but these simple ones will use our color palette:

Screenshot 2024-04-10 at 7 18 02 PM

### CODE REVIEW

File: website/src/app/kb/administer/logs/readme.mdx [hunk 1]
```diff
@@ -19,16 +21,53 @@ by advanced users and admins.
...
 ## Clearing logs
 
 Simply delete the files in the log directories listed above.
 
 For macOS, iOS, Android/ChromeOS, and Windows Clients, log directories can be
 cleared in the "Diagnostic Logs" tab in the Settings screen.
+
+```
+
+```
```
Comment (bmanifold) [inline website/src/app/kb/administer/logs/readme.mdx]: Was this intentional?

Comment (jamilbk) [reply website/src/app/kb/administer/logs/readme.mdx]: Definitely not, dang AI

Comment (jamilbk) [reply website/src/app/kb/administer/logs/readme.mdx]: fixed

**Review by bmanifold — APPROVED** [review]
LGTM

**Pull request merged**